src/schema/schema.ts

```diff
@@ -225,4 +226,9 @@ export default new GraphQLSchema({
       viewer: Viewer,
     },
   }),
+  // These are for orphaned types which are types which should be in the schema,
+  // but can’t be discovered by traversing the types and fields from query.
+  //
+  // In this case, the interface "Offer" is exposed everywhere, but the underlaying type BuyOrder needs to exist 
+  types: [BuyOrderType],
```

Comment:
I don’t understand this quite yet, but that’s no blocker. If you have any pointers to docs/source/tests, that would be much appreciated.

Reply:
Sorry I was driving to the garden state... yes it's same issue as what we saw on the Exchange version of the same PR where if a type is not referenced anywhere it doesn't end up in the schema definition. In Ruby Graphql we solved this by setting _orphan types_ but here looks like we need to explicitly add them to schema.

Reply:
Ah gotcha, thanks 👍